letsencrypt/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Let's Encrypt",
+    "version": "12.0.2.0.0",
+    "author": "Therp BV,"
+              "Tecnativa,"
+              "Acysos S.L,"
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Hidden/Dependency",
+    "summary": "Request SSL certificates from letsencrypt.org",
+    "depends": [
+        "base_setup",
+    ],
+    "data": [
+        "data/ir_config_parameter.xml",
+        "data/ir_cron.xml",
+        "views/res_config_settings.xml",
+    ],
+    "demo": [
+        "demo/ir_cron.xml",
+    ],
+    "post_init_hook": "post_init_hook",
+    "installable": True,
+    "external_dependencies": {
+        "python": [
+            "acme",
+            "cryptography",
+            "dns",
```

Comment:
I believe Travis is red because of this pip dependency: [no "dns" package was found](https://app.travis-ci.com/github/OCA/server-tools/jobs/522015141#L564)

Reply:
That package is imported as `dns` but installed as [`dnspython`](https://pypi.org/project/dnspython/). Is something trying to install the module names in the manifest as if they're package names?